src/Behat/Testwork/ServiceContainer/Configuration/ConfigurationLoader.php

```diff
@@ -195,7 +197,20 @@ protected function loadFileConfiguration($configPath, $profile)
         }
 
         $basePath = rtrim(dirname($configPath), DIRECTORY_SEPARATOR);
-        $config = (array) Yaml::parse(file_get_contents($configPath));
+
+        if (
+            \str_ends_with($configPath, '.php')
+            || \str_ends_with($configPath, '.php.dist')
+        ) {
+            $phpConfig = require $configPath;
+            if (!$phpConfig instanceof ConfigInterface) {
+                throw new ConfigurationLoadingException(sprintf('Configuration file `%s` must implements `%s`.', $configPath, ConfigInterface::class));
```

Comment:
I think this error message would be slightly clearer (and replaces a typo in the original)
```suggestion
                throw new ConfigurationLoadingException(sprintf('Configuration file `%s` must return an instance of `%s`.', $configPath, ConfigInterface::class));
```